libs/core/notification/notification-footer/notification-footer.component.ts

```diff
@@ -1,13 +1,54 @@
-import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { ChangeDetectionStrategy, Component, input, signal, ViewEncapsulation } from '@angular/core';
+import { FD_NOTIFICATION_FOOTER } from '../token';
 
 @Component({
     selector: 'fd-notification-footer',
-    template: `<ng-content></ng-content>`,
+    template: `<ng-content></ng-content>
+        @if (showTrigger()) {
+            <a role="button" class="fd-link fd-notification__link" tabindex="0" (click)="onTriggerClick()">
+                <span class="fd-link__content">{{ expanded() ? lessLabel() : moreLabel() }}</span>
+            </a>
+        } `,
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush,
     standalone: true,
     host: {
         class: 'fd-notification__footer'
-    }
+    },
+    providers: [
+        {
+            provide: FD_NOTIFICATION_FOOTER,
+            useExisting: NotificationFooterComponent
+        }
+    ]
 })
-export class NotificationFooterComponent {}
+export class NotificationFooterComponent {
+    /**
+     * Controls the visibility of the expand/collapse button.
+     * Set to `true` when the title or description is truncated or the content is expanded.
+     */
+    showTrigger = signal(false);
+
+    /**
+     * Indicates whether the content is currently expanded.
+     * When `true`, the full title and description are shown;
+     * when `false`, they are truncated after the second line.
+     */
+    expanded = signal(false);
+
+    /**
+     * Custom label for the expand button.
+     * Defaults to `'More'`.
+     */
+    moreLabel = input('More');
```

Comment:
I think the built in default will need to be translated. This is now done a bit more easily with `resolveTranslationSignalFn()`